databases/iplike/Portfile

```diff
@@ -14,20 +15,19 @@ long_description    C implementation of OpenNMS's plpgsql iplike function, \
                     preferred as it offers better performance.  Allows for \
                     complex IP address queries on a text field.
 
-homepage            http://www.opennms.org/wiki/IPLIKE
+depends_lib         port:postgresql16
 
-master_sites        sourceforge:opennms
-
-depends_lib         port:postgresql90
-
-depends_run         port:postgresql90-server
-
-checksums           rmd160  167d520c82824b1b3c6cc4ea0799bafa4cab70c4 \
-                    sha256  52703c5186ab2f8c5de86214a40ebfde3eb29454f7695ed84fa5b5d257c09eef \
-                    size    358568
-
-configure.args      --with-pgsql=${prefix}/lib/postgresql90/bin/pg_config
+fetch.type          git
+post-fetch {
+    system -W ${worksrcpath} "git submodule update --init"
+}
+checksums           rmd160  6473c366af98b0e9c4a6f45b5a97795c3a54309a \
+                    sha256  50dfe6fa7a5b210287f794f778911f4060304f19ba7a32d8af238147a2256596 \
+                    size    17278
```

Comment:
using a `git` checkout shouldn't need `checksums` - correct? (In fact, that's one of the main issues with this approach...)

Reply:
I can remove this